### ISSUE TITLE
Vector/SVG Compatibility

### ISSUE THREAD

**Issue by lquessenberry** [Jasonette/JASONETTE-iOS#6]
I'm heavy on the HTML5 CSS3 bandwagon and love using SVG graphics for backgrounds as well as icons. Can you give a heads up or documentation on using SVG graphics or how to implement vector graphics so I can get a grasp of it?

**Comment by gliechtenstein** [comment]
I just looked this up and here's the first result I found: https://www.quora.com/Should-you-use-SVG-files-for-images-in-an-iOS-app

It looks like it is possible to implement but nobody recommends it. That said, if anyone is willing to [write a component extension](https://jasonette.github.io/documentation/advanced/#extension) using something like [SVGKit](https://github.com/SVGKit/SVGKit) and share, maybe we can all try it out and decide whether to incorporate into the main project.

Also, please share if you are aware of any example of SVG working well on iOS.

So anyway, I'll keep this open, waiting for the prince charming who will step forward and build one for us :)

**Comment by lquessenberry** [comment]
Fair enough. I come from a Drupal and Responsive design background, so I just assumed SVG was kosher. That being said, how would one go about using CAShapeLayers instead?

**Comment by gliechtenstein** [comment]
That sounds interesting! I think it's definitely possible. Currently my personal priority is to work on the core part and turning the Javascript files into their own repos, so won't be able to get to it anytime soon, but if you or anyone else can contribute to writing an extension, will be delighted to try it out and share with the community. (Writing an extension is easy :) )
